contracts/package.json

```diff
@@ -77,12 +84,12 @@
     "eslint-plugin-no-only-tests": "^3.1.0",
     "ethereum-waffle": "^4.0.10",
     "ethers": "^5.4.6",
-    "hardhat": "^2.14.1",
+    "hardhat": "^2.22.17",
```

Comment:
Upgrading to this version will break the hardhat tracer for you. If you want to use that with BNB I recommend using the old hardhat versions.

Reply:
Have reverted this change